core/src/main/javascript/reportviewer/reportviewer.js

```diff
@@ -818,7 +818,7 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
         if(isRunningIFrameInSameOrigin) {
           if (!top.mantle_initialized) {
             this._topMantleOpenTabRegistration = top.mantle_openTab = function(name, title, url) {
-              window.open(url, '_blank');
+              window.open(xssUtil.sanitizeUrl(url), '_blank');
```

Comment:
Should use the new `xssUtil.openWindow(..)` method instead.

Reply:
Updated